DelvUI/Interface/Jobs/SamuraiHud.cs

```diff
@@ -239,15 +254,15 @@ public class SamuraiConfig : JobConfig
         [CollapseControl(30, 0)]
         public bool ShowKenkiBar = true;
 
-        [DragFloat2("Kenki Bar Size", max = 2000f)]
+        [DragFloat2("Kenki Bar Size" + "##Kenki", max = 2000f)]
```

Comment:
For me that, appending "##Kenki", would make sense if there was multiple fields named "Kenki Bar Size" and CategoryField.ID == null (which sadly currently is for the cases which i checked, dynamicList, DragFloat2).

If CategoryField.ID != null the only points when appending ImGui comment to the friendlyName is when you have multiple fields on the same level with the same name.

Reply:
ooooff  i feel that explained it poorly. but basically:
if the friendlyName is "Kenki Bar Size", there is no point to append "##Kenki".
if the firendlyName is "Size", there you have a reason to append "##Kenki" to the name due to IDText() being null for for fields who are not in NestedConfig.